lib/common/tls.c

```diff
@@ -216,16 +216,26 @@ pcmk__init_tls(pcmk__tls_t **tls, bool server, gnutls_credentials_type_t cred_ty
             gnutls_anon_allocate_client_credentials(&(*tls)->credentials.anon_c);
         }
     } else if (cred_type == GNUTLS_CRD_CERTIFICATE) {
-        /* Grab these environment variables before doing anything else. */
-        if (server) {
-            (*tls)->ca_file = pcmk__env_option(PCMK__ENV_CA_FILE);
-            (*tls)->cert_file = pcmk__env_option(PCMK__ENV_CERT_FILE);
-            (*tls)->crl_file = pcmk__env_option(PCMK__ENV_CRL_FILE);
-            (*tls)->key_file = pcmk__env_option(PCMK__ENV_KEY_FILE);
-        } else {
+        /* Try the PCMK_ version of each environment variable first, and if
+         * it's not set then try the CIB_ version.
+         */
+        (*tls)->ca_file = pcmk__env_option(PCMK__ENV_CA_FILE);
```

Comment:
It's probably too late since I didn't see this until after 3.0.0 was released... but why did we bother with two different environment variables? Seems like we could have used `PCMK_x` instead of `CIB_x` in all cases, since there were no backward compatibility concerns at the time